challenges.tex

```diff
@@ -0,0 +1,540 @@
+so getting only partial credits for a challenge disqualifies from getting the
+bonus points for the fastest completion.
+
+\subsection{Changes compared to the Main Competition}
+The tasks covered in the various challenges mostly obey the regular rules
+for the \ac{RCLL}. However, some aspects are altered to simplify the setup.
+The changes are not affecting existing machine communication and processing
+steps, such that the challenges can be carried out on real machines as well as
+on mockup versions obeying the requirements outlined in \refsec{sec:machines}.
+
+\paragraph{Product Delivery}
+The delivery procedure for finished products is altered compared to the
+\ac{RCLL} rule set. In order to reduce the amount of machines required
+for participation, Deliveries are made by bringing the finished product
+to the insertion zone and dropping it there.
+
+\paragraph{Ring Payments}
+Easing the setup of mockup machines, it is not required to have a slide
+on ring stations. Instead, a shelf or additional conveyor may be mounted on the
+machine to place payments at the corresponding station.
+
+\paragraph{Ring Color Assignment}
+The cost for mounting each ring color are fixed, the assignment of ring colors
+is semi-fixed as teams can choose between two different options for each
+challenge (\texttt{option1} or \texttt{option2} according to
+\reftab{tab:ring-costs}).
+The RefBox settings default to \texttt{option1}, but teams may change this
+accordingly.
+
+\newcommand{\colconfig}{\mathcal{RC}}
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l||l|l||l|l}
+  & \multicolumn{4}{c||}{Ring Costs}
+  & \multicolumn{2}{c}{\multirow{2}{*}{Color Assignment }}\\\cline{2-5}
+  & Color  & Price & Color  & Price & \multicolumn{2}{c}{}\\\cline{2-7}
+  & Yellow & 0 & Green & 0
+  & RS1: $\colconfig_1$ & RS1: $\colconfig_2$ \\
+  & Blue  & 1 & Orange & 2
+  & RS2: $\colconfig_2$ & RS2: $\colconfig_1$ \\\hline\hline
+  Configuration & \multicolumn{2}{c||}{$\colconfig_1$}
+  & \multicolumn{2}{c||}{$\colconfig_2$}
+  & $\texttt{option1}$ & $\texttt{option2}$\\
+ \end{tabular}
+ \caption{Materials}
+ \label{tab:ring-costs}
+\end{table}
+
+\paragraph{Materials}\label{sec:materials}
+The available material that can be used per challenge is restricted
+(unless stated otherwise) per machine according to the information in
+\reftab{tab:materials}.
+\begin{table}[!htb]
+ \centering
+  \begin{tabularx}{\linewidth}{l|l}
+   Machine & Available Material  \\\hline
+   \ac{BS} & 2 bases of each color \\
+   \ac{CS} & 3 cap-carriers (cap color choices up to each team)  \\
+   \ac{RS} & 4 rings of each assigned color (8 in total)  \\
+  \end{tabularx}
+ \caption{Materials}
+ \label{tab:materials}
+\end{table}
+
+\paragraph{Orders}
+Unless specified otherwise, orders that have to be fulfilled in challenges
+are entered through the web shop
+(see the RefBox workshop in \refsec{sec:qualification})
+by any member of the competing team.
```

Comment:
What is the competing team? IIUC, there is not (one) opponent, because each team runs the challenges by itself. So who exactly can do this?